sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp

```diff
@@ -6424,11 +6424,14 @@ namespace Azure { namespace Storage { namespace Blobs {
           {
             response.Details.HttpHeaders.CacheControl = cache_control__iterator->second;
           }
-          auto content_md5__iterator = httpResponse.GetHeaders().find("content-md5");
-          if (content_md5__iterator != httpResponse.GetHeaders().end())
+          if (http_status_code == 200)
```

Comment:
what happens for non 200 cases ? should we at least log something, it looks like we are swallowing non 200 responses, is that the desired behavior?

Reply:
For 206 responses, blob hash is returned in `x-ms-blob-content-md5` header, which is parsed at L6436